sdk/cosmos/azure-cosmos/azure/cosmos/_location_cache.py

```diff
@@ -116,12 +163,43 @@ def perform_on_database_account_read(self, database_account):
             database_account._EnableMultipleWritableLocations,
         )
 
-    def get_ordered_write_endpoints(self):
+    def get_ordered_write_locations(self):
         return self.available_write_locations
 
-    def get_ordered_read_endpoints(self):
+    def get_ordered_read_locations(self):
         return self.available_read_locations
 
+    # This updates the current and previous of the regional endpoint
+    # to keep it up to date with the success and failure cases
+    def swap_regional_endpoint_values(self, request):
```

Comment:
Is this method being used? I could see it being called from `global_endpoint_manager#swap_regional_endpoint_values` but no usages from there.